src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -122,9 +122,12 @@ private static string GetFolderPathCoreWithoutValidation(SpecialFolder folder)
                 Debug.Fail($"Unable to get home directory: {exc}");
             }
 
+            // Like mono, fall back to '/' when we can't determine the home directory.
```

Comment:
Let's remove the "like mono"... hopefully eventually this code will be used by mono :)